src/IO/lbsLoadReaderVT.py

```diff
@@ -191,67 +203,74 @@ def json_reader(self, returned_dict: dict, file_name: str, phase_ids, node_id: i
             except brotli.error:
                 decompressed_dict = json.loads(compr_bytes.decode("utf-8"))
 
-        # validate schema
+        # Validate schema
         if SchemaValidator().is_valid(schema_to_validate=decompressed_dict):
             print(bcolors.HEADER
                   + "[LoadReaderVT]"
                   + bcolors.END
-                  + "Valid JSON schema in  {}".format(file_name))
+                  + " Valid JSON schema in  {}".format(file_name))
         else:
             raise SyntaxError(f"{bcolors.ERR}[LoadReaderVT] Invalid JSON schema in {file_name}{bcolors.END}")
 
-        # defining phases from file
+        # Define phases from file
         phases = decompressed_dict["phases"]
-        comm_dict = dict()
-        # case for empty Rank
+        comm_dict = {}
```

Comment:
This is implicit

Reply:
Nope, it's explicit and more efficient perfomance-wise
```
>>> a={}
>>> type(a)
<class 'dict'>
>>> b={1,2,3}
>>> type(b)
<class 'set'>
```
http://katrin-affolter.ch/Python/Python_dictionaries